### ISSUE TITLE
Warning messages in log on startup (not urgent)

### ISSUE THREAD

**Issue by Andy2No** [christofmuc/KnobKraft-orm#185]
When I start 1.17.0 and look at the log, I get this:

05:57:14: Overriding built-in automatic category rules with file C:\Users\andy\AppData\Roaming\KnobKraft\automatic_categories.jsonc
05:57:14: Overriding built-in import category rules with file C:\Users\andy\AppData\Roaming\KnobKraft\mapping_categories.jsonc
05:57:14: Adaptation: Warning: file name Behringer Deepmind 12 is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name DSI Pro 2 is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name DSI Prophet 08 is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name DSI Prophet 12 is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name Matrix 6 is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name PioneerToraiz-AS1 is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name Roland JX-8P is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name Sequential Pro 3 is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name Sequential Prophet 5 Rev4 is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name Sequential Prophet 6 is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name Sequential Prophet X is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name Studiologic Sledge is not a valid module identifier in Python, please use only lower case letters and numbers
05:57:14: Adaptation: Warning: file name Waldorf Blofeld is not a valid module identifier in Python, please use only lower case letters and numbers

Perhaps this is due to the database format no longer being fully backwards compatible?  Or maybe due to the ,jsonc files?  I only set those up to try the features, which I still don't really understand, so I'm happy to delete them, if that helps.

**Comment by christofmuc** [comment]
@Andy2No Ah, you're right. I had ignored these warnings as they come when you set the user adaptation directory to be the source code's adaptation directory. But with 1.17.0 the adaptations are no longer compiled into the executable, but shipped as separate Python files in an adaptation subdirectory. This makes it easier for them to reference each other, e.g. for the XV-3080 to import the JV-1080, because it is fully backwards compatible. 

The warning is correct, in that the files ideally should not have spaces in their names. I refrained from fixing so far, but I think I should do it and hope it doesn't cause any problems.

**Comment by christofmuc** [comment]
I decided to disable the warning with the next release, as these files are actually read ok. I think at some point I built in a identifier cleaner or something that actually allows the file to be used, and the warning was old. Renaming those files to have no spaces in them sounds much more dangerous, as I would need to remove the old files during installation, and what KnobKraft doesn't like is two files with the same synth name.
